src/test/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientImplTest.java

```diff
@@ -49,8 +49,9 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonParseException;
 
-import static org.junit.Assert.assertEquals;
```

Comment:
While it's nice that you added these tests, I'd rather completely revert these changes. I actually created new unit tests to test the concurrency of the client: these tests will fail if the HTTP entities are not consumed, without having to rely on checking implementation details.

Reply:
Much better indeed. Other test changes have been reverted.